libraries/app/include/graphene/app/api.hpp

```diff
@@ -534,12 +534,12 @@ namespace graphene { namespace app {
          /**
           * @brief Get all stored objects of an account in a particular catalog
           *
-          * @param account Account name to get info from
+          * @param account Account name or ID to get info from
```

Comment:
In an earlier PR I changed all "name or ID" instances to "ID or name" for consistency. :)